File: KuantumKaos/python/python_kaos/veri_paketi.py
from kuantum_nesnesi import KuantumNesnesi
from kuantum_cokusu_exception import KuantumCokusuException

class VeriPaketi(KuantumNesnesi):
    def analiz_et(self):
        self.stabilite -= 5
        print("Veri paketi analiz edildi.")

        if self.stabilite <= 0:
            raise KuantumCokusuException(self.id)

File: KuantumKaos/java/kuantumKaosJava/src/VeriPaketi.java
public class VeriPaketi extends KuantumNesnesi {

    @Override
    public void analizEt() {
        stabilite -= 5;
        System.out.println("Veri paketi analiz edildi.");

        if (stabilite <= 0) {
            throw new KuantumCokusuException(id);
        }
    }
}
